src/reqstool/resources/schemas/v1/common.schema.json

```diff
@@ -217,6 +217,11 @@
         "reqid": {
             "type": "string"
         },
+        "urnid": {
+            "type": "string",
+            "pattern": "^[^:].*",
+            "description": "Field that can either be written with a system prefix like 'SYSTEM:ID' or just like an id, omitting the system prefix if the id is located in the same system as the current file you are writing in."
```

Comment:
Suggestion:

An identifier that can be on the form '<system urn:id>' or '<id>'. If the system urn is omitted then it defaults to the system urn that the the current file is part of.

Reply:
Great! Thanks!